cmd/fluxd/main.go

```diff
@@ -118,6 +116,7 @@ func main() {
 	var clusterVersion string
 	var sshKeyRing ssh.KeyRing
 	var k8s cluster.Cluster
+	var k8s_inst *kubernetes.Cluster
```

Comment:
Might be a bit more inclusive of hypothetical alternate cluster implementations to have a variable of type `() -> []ImageCreds` to be assigned. A nit.